spec/features/admin/admin_can_add_items_spec.rb

```diff
@@ -6,6 +6,7 @@
     @user = User.create(full_name:"Andrew Fink", email:"andrewfink@gmail.com", password:"password", role: 1)
     ApplicationController.any_instance.stub(:current_user).and_return(@user)
     @item = Item.create(title: "chicken-leg", description: "yum", price: 40000)
+    @item.categories.create(name: 'test menu')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.